pkg/registry/apis/iam/resourcepermission/sql.go

```diff
@@ -1,11 +1,228 @@
 package resourcepermission
 
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/grafana/authlib/types"
+
+	"github.com/grafana/grafana/apps/iam/pkg/apis/iam/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/iam/legacy"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/sqlstore/session"
+	"github.com/grafana/grafana/pkg/storage/legacysql"
+)
+
 // List
 
 // Get
 
 // Create
 
+// TODO: use mapper?
```

Comment:
is there an existing mapper to use? or wdym?

Reply:
Yes, once the get PR is merged.